dialogue/TownNPCs/ShovelGuy/ShovelGuy1.json

```diff
@@ -14,7 +14,7 @@
       "And it involves my back."
     ],
     "pl": [
-      "<c=mocha>Mocha</c> poprosiła mnie, żebym wykopał trochę miejsca na magazynę, który ona planuje zbudować.",
+      "<c=mocha>Mocha</c> poprosiła mnie, żebym wykopał trochę miejsca na magazyn, który ona planuje zbudować.",
       "Znamy się od dawna, więc nie miałem problemu pomagając starej przyjaciółce!",
```

Comment:
"z pomocą" will fit here better than "pomagając".

Reply:
I don't really think "za pomocą" works here. That phrase means "with the help of" instead of "helping". That would turn the clause into "nie miałem problemu za pomocą starej przyjaciółce", meaning "i had no problem with the help of an old friend".